test/shared/commonFunctionsForTests.sh

```diff
@@ -382,36 +453,40 @@ function gcclassic_config_options() {
 }
 
 
-function gcclassic_compiletest_name() {
+function compiletest_name() {
     #========================================================================
     # Returns the GCClassic configuration options given a directory name
     #
-    # 1st argument: Directory name
+    # 1st argument: Model name ('gcclassic' or 'gchp')
+    # 2nd argument: Build directory name
     #========================================================================
+    model="${1}"
+    buildDir=$(basename "${2}")   # Only take last part of path
 
-    # Arguments
-    dir=$(basename ${1})   # Only take last part of path
+    # Display the proper model name
+    [[ "x${model}" == "xgcclassic" ]] && displayName="GCClassic"
+    [[ "x${model}" == "xgchp"      ]] && displayName="GCHP"
 
     # Turn on case-insensitivity
     shopt -s nocasematch
 
     # Pick the proper build options
-    if [[ ${dir} =~ "apm" ]]; then
-	result="GCClassic with APM"
-    elif [[ ${dir} =~ "carbon" ]]; then
-	result="GCClassic with carbon (as a KPP mechanism)"
-    elif [[ ${dir} =~ "luowd" ]]; then
-	result="GCClassic with Luo et al wetdep"
-    elif [[ ${dir} =~ "hg" ]]; then
-	result="GCClassic with Hg (as a KPP mechanism)"
-    elif [[ ${dir} =~ "rrtmg" ]]; then
-	result="GCClassic with RRTMG"
-    elif [[ ${dir} =~ "tomas15" ]]; then
-	result="GCClassic with TOMAS15"
-    elif [[ ${dir} =~ "tomas40" ]]; then
-	result="GCClassic with TOMAS40"
+    if [[ "${buildDir}" =~ "apm" ]]; then
+	result="${displayName} with APM"
+    elif [[ "${buildDir}" =~ "carboncycle" ]]; then
+	result="${displayName} with carboncycle (as a KPP mechanism)"
```

Comment:
Change to "carbon" in both lines

Reply:
Now fixed in commit da258592

Reply:
Fixed in commit da25859232112f5bad1c2e3ad235db0c71048620